### ISSUE TITLE
Web Mercator Weirdness

### ISSUE THREAD

**Issue by emackey** [AnalyticalGraphicsInc/cesium#3153]
I tried enabling WebMercator mode like this:

```
var viewer = new Cesium.Viewer('cesiumContainer', {
    mapProjection: new Cesium.WebMercatorProjection(Cesium.Ellipsoid.WGS84)
});
//viewer.extend(Cesium.viewerCesiumInspectorMixin);
viewer.scene.morphTo2D(0);
```

But the world went awash in wacky weirdness:

![badmercator_v2](https://cloud.githubusercontent.com/assets/1235080/10854547/c5127544-7f12-11e5-894d-f0bae6ff9ddc.gif)

Is this a bug or am I just doing it wrong?  If it's not a bug then we might need clearer docs on how to do this.

/cc @kring @abwood

**Comment by pjcozzi** [comment]
Can you reproduce this in 1.14?

**Comment by emackey** [comment]
This is master 595bb4e9f426 from Oct 26.

**Comment by emackey** [comment]
I'll check 1.14

**Comment by emackey** [comment]
Yeah, this behavior goes way back, I can reproduce in 1.1 and 1.14

**Comment by kring** [comment]
@emackey I remember seeing these kinds of artifacts ages ago, and I thought at the time that it was a result of projecting Bing tiles from Web Mercator to Geographic and then back to Web Mercator; we're losing information.  There's a roadmap item to fix this (https://github.com/AnalyticalGraphicsInc/cesium/issues/526 search for mercator), but doing so probably requires us to keep the original Web Mercator images around "just in case."  Although, this looks a bit worse than I remember, so maybe there is something more nefarious going on.

**Comment by mramato** [comment]
Could #3104 have cause this to be more noticeable?

**Comment by emackey** [comment]
It's possible that WebMercator just isn't used much.  I had to go back to b28 to find a Sandcastle example of it, and it appeared to exhibit this bug there too.  My main concern isn't blurriness, it's the noticable "pop" where the top half of Greenland  moves several degrees North or South as you cross a tile/zoom boundary.  I don't think the re-projection back from Geographic to Mercator is working, at least not for texels between the vertices (try wireframe mode in Cesium Inspector to see this).

**Comment by emackey** [comment]
You can just barely see in the wireframe version, but the Northern coast of Greenland shifts a noticable amount here.  The same is happening in the other direction near the South Pole.  I'm suspicious that the polar "caps" are playing a role here too.

![badmercator_wireframe](https://cloud.githubusercontent.com/assets/1235080/10895480/77c57b3e-8182-11e5-91a2-1236650d602f.gif)

**Comment by cfickler** [comment]
Could this be related to the issue I am seeing with stretching of the map when the height of the canvas exceeds the width of the canvas, you are viewing the full world and you zoom out using the mouse scroll ? This is not a polar problem either as you can rotate the map 90 degress and watch is stretch east and west. Also, slightly changing the size of the canvas (via the window) sets it back to what you would expect. This seems to happen in both Web Mercator and WGS84

Stretched Map
![image](https://cloud.githubusercontent.com/assets/16298736/11800830/403cdf94-a334-11e5-9b44-430623219c25.png)

After slightly changing the window size
![image](https://cloud.githubusercontent.com/assets/16298736/11800841/620617d0-a334-11e5-9a5d-a883c8473ba6.png)

**Comment by mramato** [comment]
@cfickler , I think what you are seeing is an unrelated issue, so I submitted #3346 for it.

**Comment by samcityusman-hub** [comment]
Clear bug
